_clubs/klahanie-swim-team.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Klahanie Sharks Swim Team
+date: 2019-06-10 07:36
+author: Klahanie
+comments: true
+link: true
+categories: [pools, clubs]
+show-title: true
+banner: /images/shark-swim-banner.gif
+thumbnail: /images/shark-swim-banner.gif
+description: The Klahanie Sharks are a fun swim league for kids aged 8 to 18!
+---
+The Klahanie Association is proud to partner with the the Klahanie Swim Team to provide a great opportunity for Klahanie youth to participate in team sports. The Klahanie Swim Team is a member of the 26-Team, Midlakes Summer Swim League. We seek to provide a competitive swimming environment for a young swimmer that is fun and relaxed.
+
+Klahanie Swim Team offers swimmers between 5 and 18 years of age a fun way to get exercise while swimming with others of the same age. Although swimming is an individual sport, participating on a swim team helps create an enthusiasm for swimming that is hard to achieve by just swimming alone. The Klahanie Swim Team promotes team building skills, friendliness, and sportsmanlike conduct among teammates and opponents.
+
+Klahanie Swim Team is a competition where swimmers race at meets against other swimmers of their own age group. The swim meets emphasize fun and personal improvement.  They team that success isn’t necessarily winning, but is related to hard work and doing your best. The Klahanie Swim Team helps teach that competition, winning, and losing are all part of life. There are 5 boys and girls age brackets: 8, 10, 12, 14 and under, and 15 and over.
+
+The Klahanie Homeowners Association is equipped with two, four-lane, 25-meter competition pools. Practices are held daily from Memorial Day through July. Klahanie Swim Team coaches and lifeguards are also available throughout the swim season for private, semi-private and group lessons.

```

Comment:
Recommend a rewrite for this page, paying special to when or when not to spell out numerals.
"private, semiprivate, and group lessons."

Reply:
Thanks, feel free to take a cut at this. Instructions on how to make an edit are here: https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository